lib/utils/GetImageURL.tsx

```diff
@@ -0,0 +1,13 @@
+export const getImageURL = (
+  el: CanvasImageSource,
+  { width, height }: { width: number; height: number },
+) => {
+  const canvas = document.createElement('canvas')
+
+  canvas.width = width
+  canvas.height = height
+
+  canvas?.getContext('2d')?.drawImage(el, 0, 0, width, height)
```

Comment:
We should add a comment about which browsers this is expected to work on (and which ones we tested).

Reply:
Done